src/oge/load_data.py

```diff
@@ -490,19 +490,27 @@ def update_epa_to_eia_map(cems_df: pd.DataFrame) -> pd.DataFrame:
     return cems_df
 
 
-def add_report_date(df: pd.DataFrame) -> pd.DataFrame:
+def add_report_date(
+    df: pd.DataFrame, plant_timezone: pd.DataFrame | None = None
+) -> pd.DataFrame:
     """Add a report date column to the CEMS data frame based on the plant's local
     timezone
```

Comment:
I guess it applies to all data frame with the required columns. Not only CEMS. Or we should rename the function to `add_report_date_to_cems_df`

Reply:
Updated